exam/src/main/java/in/testpress/exam/ui/ReviewStatsFragment.java

```diff
@@ -313,24 +316,23 @@ private void showOrHidePercentageLayout() {
     }
 
     private void showOrHideScoreLayout() {
-        if ((exam.getShowScore()) && (attempt.getScore() == null || attempt.getScore().equals("NA"))) {
-            scoreLayout.setVisibility(View.GONE);
-        } else {
+        if ((exam == null || exam.getShowScore()) && attempt.getScore() != null && !attempt.getScore().equals("NA")) {
```

Comment:
`attempt.hasScore`
 This should check if score is not null and not equal to "NA". This can be directly committed to master